Metro-Records/la-metro-councilmatic
# Store custom biography and headshot on Councilmatic model, not OCD model

## Overview

See title. It also upgrades django-councilmatic to capture a relevant upstream change, removes unused copies of the headshot and bio form templates, and updates some GHCR image references needed after we moved repositories and renamed the master branch to main.

Connects #957 

## Testing Instructions

 * Pull down these changes and rebuild your local app: `docker-compose build app`
 * Create a super user, if needed: `docker-compose run --rm app python manage.py createsuperuser`
 * Bring up the app: `docker-compose up app`
 * Navigate to Katy Y.'s board member page and add a bio and headshot. Confirm they show up as expected.
 * Run a person scrape: `docker-compose run scrapers pupa update lametro --rpm=0`
 * Hard refresh Katy Y.'s board member page and confirm her bio and headshot are still present.
